contracts/Forwarder.sol

```diff
@@ -62,6 +62,21 @@ contract Forwarder {
     flush();
   }
 
+  /**
+   * ERC721 standard callback function for when a ERC721 is transfered. The forwarder will send the nft
+   * to the base wallet once the nft contract invokes this method after transfering the nft.
+   *
+   * @param _operator The address of the nft contract
+   * @param _from The address of the sender
+   * @param _tokenId The token id of the nft
+   * @param _data Additional data with no specified format, sent in call to `_to`
+   */
+  function onERC721Received(address operator, address from, uint256 tokenId, bytes memory data) public virtual override returns (bytes4) {
+    IERC721(_operator).safeTransferFrom(from, parentAddress, tokenId, data);
```

Comment:
using safeTransferFrom because we know the nft contract implements it since this callback has been invoked